src/debug/sdk_path.ts

```diff
@@ -0,0 +1,30 @@
+"use strict"
+
+// This file exists to write the SDK path from config in the extension and read it from the
+// debug adapter. Everything imported here must be available in both places (eg. no vscode).
+import * as os from "os";
+import * as path from "path";
+import * as fs from "fs";
+
+const sdkPathStoreName = "dart-code.sdk";
+const sdkPathStore = path.join(os.tmpdir(), sdkPathStoreName);
```

Comment:
Since it's just throwaway info, I think temp is better.. I hate my home folder being clogged up with crap from apps; it's *my* home folder! IMO if I don't know what it's for, it shouldn't be in my home folder!. I don't feel too strongly if you think there are reasons to put elsewhere though (but having what looks like another settings file separate to the VS one might seem weird to the user).